### ISSUE TITLE
Donations

### ISSUE THREAD

**Issue by yan12125** [lxqt/lxqt#1846]
##### Expected Behavior
LXQt should be able to accept donations

##### Current Behavior
I didnt find it

##### Possible Solution

First we need a legal entity. From https://github.com/lxqt/lxqt/issues/1829#issuecomment-692002460

> I'm not 100% sure about it, but from a legal point of view I think to collect donations there has to be at minimum an association, or a company or similar - a legal personality, if donations go to an indivudual person there are taxable as income.

I remember some time ago donations went to a company owned by one of KeePassXC maintainer. Seems that is no longer the case, though.

Here is another example: donations for the Arch Linux project are via Software in the Public Interest (SPI) Inc. [1] Maybe there will not be taxes with that approach?

I see some projects have a detailed list on how money is spent. For example, Manjaro hosted their list on [2]. That would be nice if LXQt has something similar.

[1] https://www.spi-inc.org/projects/archlinux/
[2] https://opencollective.com/manjaro/expenses

##### Steps to Reproduce (for bugs)
1. Be generous and give money to LXQt

##### Context
Maintaining a server needs money. That includes domain name fee, VPS fee, ...

##### System Information
* Distribution & Version: Read World, version 2020

**Comment by stefonarch** [comment]
I've had a look in SPI https://www.spi-inc.org/projects/services/ it sound interesting but IMO would not solve the problem of who can  receive finally the donations which should be legal persons:

>  Once held by SPI, funds may only be transferred to another 501(c)3 US non-profit with a compatible purpose. Examples of such non-profits include the Apache Software Foundation, the Software Freedom Conservancy, and the Free Software Foundation.

More interesting could be opencollective, EU-based, they seem to have an easier way  called "fiscal host" for the legal person.
https://opencollective.com/hosts

> Hosts are legal entities that collect money on behalf of open collectives so that they don't have to worry about accounting, taxes, etc. Some also provide extra services.

https://docs.opencollective.com/help/fiscal-hosts/become-a-fiscal-host

For LXQt mostly  one of those would be the "umbrella":
https://opencollective.com/opensource#section-contributions  (US)
https://opencollective.com/europe (EU)

There is also UK based, but manjaro is EU.

> When do I need to worry about VAT?
If your collective plans to offer products and services on a regular basis, you should apply for your own VAT number. If that’s the case, we would highly recommend you to create your own fiscal host for your collective that could also act as a fiscal host for collectives like yours in your country. The more open collective hosts the more they will be able to focus on a certain type of collectives and be in a position to offer more services to help them focus on their mission.

I think manjaro is a good example to look at. One 10k declined, bought some hardware for one person and still 6k available.
In the case of LXQt the future server costs will be probably only some low percent of incoming donations, and there has to be decided where to spend them.

**Comment by shlyakpavel** [comment]
> and there has to be decided where to spend them.
I suggest ordering cookies 🥠

**Comment by yan12125** [comment]
The opencollective project looks quite promising, so I created https://opencollective.com/lxqt. @stefonarch if you also find it satisfying, could you check how to get paid for Hetzner services?

> and there has to be decided where to spend them.

Some ideas: more powerful servers; pay @agaida fees for lxqt.org, etc.

**Comment by stefonarch** [comment]
Ok, I made a login and somehow I arrived here:

> Request payment from LXQt for work you’ve done or to be reimbursed for purchases. Expenses will be processed for payment once approved by a Collective admin. Only the amount and description are public in the Collective’s transparent budget—attachments, payment details, and other personal info is kept private.

IMO if we devide to use it we need decide some things:

* Who will be our "Collective Admins"?
* Where goes eventual surplus? 
 Depending on where we publish the  donation information only server costs will covered very soon, also activating backup and so on. Server cost is ~€90/year now-
 Maybe an option to donate it to some other foundation (EFF, Qt, ecc)?

**Comment by yan12125** [comment]
> Who will be our "Collective Admins"?

Maybe someone who knows accounting? Also, like all other occupations, there should be at least two Collective Admins

> Maybe an option to donate it to some other foundation (EFF, Qt, ecc)?

Donating to other organizations sounds good. Or we can list monthly budgets and tell users to donate to those other organizations when the goal has been reached. The latter avoids 10% fees from opencollective.

> More info on the site about LXQt

Sure. At least https://lxqt-project.org/ and https://github.com/lxqt/lxqt should mention it. A `FUNDING.yml` like 
https://github.com/keepassxreboot/keepassxc/blob/develop/.github/FUNDING.yml is also good to have.

**Comment by yan12125** [comment]
I dropped some money to the LXQt collective. Could you use that for the server expense next month to see if it works or not?

> Maybe someone who knows accounting?

If no other one volunteers, I will take on it in the short term. I still hope someone better than me can join.

**Comment by stefonarch** [comment]
Spam filter at riseup is too strong, found this only now together with other GH stuff... Will check soon.

**Comment by wxl** [comment]
@stefonarch account.riseup.net > mail settings > whitelist and add notifications@github.com

**Comment by stefonarch** [comment]
Yeah, I've done that just 5 min ago... filed also a payment request for January.

**Comment by yan12125** [comment]
> filed also a payment request for January.

Thanks! If everything is confirmed to work well, we can start adding donation links everywhere :)

**Comment by stefonarch** [comment]
Payout has arrived here today.

**Comment by yan12125** [comment]
Great! Looks like there is some human's checking. 10% fee is worth it.

Time to add the donation link everywhere! Maybe also include some notes about donation in 0.17 announcement.

**Comment by stefonarch** [comment]
We need a board/collective admins of at least three people to decide wether accept or not incoming requests.
I just saw this:
https://opencollective.com/lxqt#category-BUDGET

And some basic information page for what we'll do with the funds IMHO is needed too. Server cost is obvious: Btw is there any GH cost for the repo?

**Comment by telmo1241** [comment]
I don't understand accounting, but when it comes to advertising, you can count on me

**Comment by yan12125** [comment]
> We need a board/collective admins of at least three people to decide wether accept or not incoming requests.

Given there are apparently professionals at Open Collective working on checking expenses, I'd say knowledge in accounting is not needed and it is great if anyone in the LXQt team volunteers.

> Btw is there any GH cost for the repo?

From https://github.com/organizations/lxqt/settings/billing, there are no costs from GitHub for now. By the way, from that page agaida is the only person that receives billing mails from GitHub. More people should be added - maybe the same ones at Open Collective?

And from https://github.com/lxqt/lxqt/pull/1935#issuecomment-791610593,

> logo, text, information about how donations are intended to use

I uploaded the LXQt logo. I will add other details when I find time. If will be greater if someone else volunteers as an admin at Open Collective and updates project descriptions. I'm getting busier these days.

> when it comes to advertising, you can count on me

Thank for the help! Advertising can be another big issue. The part related to donations may be the News field at Open Collective. It will be great if you have suggestions :)

---

Another idea if we have extra budget: establish infeastructure for backups. Recent OVH fire incident makes me sad and feared 😢

**Comment by yan12125** [comment]
Regarding project descriptions on Open Collective, the first 3 paragraphs of https://lxqt-project.org/about/ seem good enough. Any other selling points not mentioned there?

**Comment by stefonarch** [comment]
> LXQt has already been included in most Linux and BSD distributions 

maybe better

> LXQt is available in most Linux and BSD distributions 

fot the rest it's good I think.

**Comment by telmo1241** [comment]
Would it be possible to create a non-profit lxqt foundation?
In addition, I only see the possibility of the lxqt project becoming a company

**Comment by yan12125** [comment]
> fot the rest it's good I think.

Thanks, nice catch! I updated the description at the bottom of https://opencollective.com/lxqt. Mind to submit your fixes to the lxqt.github.io repo?

> Would it be possible to create a non-profit lxqt foundation?

I'm all for it, but some people with knowledge about legal stuffs are needed. I gave up after reading the first few comments about taxes in this issue.

> I only see the possibility of the lxqt project becoming a company

Hmm, mind to share more details about your concerns?

**Comment by telmo1241** [comment]
We can follow the example of manjaro, where the project became a company

Traveling a lot already mayonnaise (But pacxra that happened would involve a lot of work) 
Create a development system similar to kde neón, endless os or gnome os, only that totally for lxqt with stable versions and development versions, so such a company could make agreements with hardware companies so that lxqt stable came preinstalled on these computers

That's just an idea.

**Comment by yan12125** [comment]
Not sure about other parts, but the idea of maintaining multiple versions (stable & development) was once mentioned but not accepted (I'm too lazy to find related discussions). Anyway those are far beyond financial issues and another thread will be better.

**Comment by telmo1241** [comment]
> Não tenho certeza sobre outras partes, mas a ideia de manter várias versões (estável e desenvolvimento) foi mencionada uma vez, mas não aceita (estou com preguiça de encontrar discussões relacionadas). De qualquer forma, essas estão muito além das questões financeiras e outro tópico será melhor.

New topic created, now i would like everyone involved in the lxqt project to give their opinion, reaching a consensus

**Comment by stefonarch** [comment]
https://manjaro.org/donate/
Similar Paragraph are needed IMHO in the Opencollective page.

**Comment by yan12125** [comment]
> Similar Paragraph are needed IMHO in the Opencollective page.

It will be great if someone can organize such statements for LXQt. I may not have enough time for that before the next release.

**Comment by yan12125** [comment]
@stefonarch I'm not sure if you got notifications from OpenCollective - we got one more financial contributor. You may want to submit server costs for February and March.

**Comment by stefonarch** [comment]
No, I don't get any. Will do.

**Comment by stefonarch** [comment]
What about activating backup at Hetzner at this point?

**Comment by yan12125** [comment]
> What about activating backup at Hetzner at this point?

+1 for backups. An issue is that https://docs.hetzner.com/robot/dedicated-server/backup/ does not indicate whether backups are stored in a different data center or not. We need to keep safe even with accidents like OVH fire.

**Comment by yan12125** [comment]
> Will do.

Approved the latest expenses and updated descriptions for usage of donations!

**Comment by stefonarch** [comment]
It's not a dedicated server, I looked into it yesterday, there are 2 options.
>SNAPSHOTS
Snapshots are instant copies of your servers disks.
You can create a new server from a snapshot and even transfer them to a different project.
We recommend that you power down your server before taking a snapshot to ensure data consistency.
Snapshots cost €0.0122/GB/month (incl. 22 % VAT).SNAPSHOTS

>BACKUPS
Backups are automatic copies of your servers disks. For every server there are seven slots for backups.
If all slots are full and an additional one is created, then the oldest backup will be deleted.
We recommend that you power down your server before creating a backup to ensure data consistency on the disks.
Enabling Backups for your server will cost 20 % of your server plan per month.

it looks like it's not automatic.

**Comment by stefonarch** [comment]
Are there different roles in "Our Team"? If so the most active members could join all, and one other admin would do.

**Comment by yan12125** [comment]
> It's not a dedicated server but backups are the same, I looked into it yesterday, there are 2 options:

Thanks for checking it! Does that mean we need to shutdown the server from time to time? Not quite convenient lol.

For my own machine, I just rsync important data to another machine. That should be cheaper than full disk backups, too.

> Are there different roles in "Our Team"? 

Not sure what you meant. Could you explain more?

**Comment by stefonarch** [comment]
> Not sure what you meant. Could you explain more?

Looking at "Team" your avatar has an "administrator" label below, so I wondered if there are also other "levels" fo team members.

As for the backups, it looks like yes, but actually I restart it also after kernel upgrades, so doing it twice monthly would be better than nothing. Better would be an rsync thing of /home and /etc to another machine.

**Comment by yan12125** [comment]
> there are also other "levels" fo team members.

Oh yep, there are also "core contributors" and "accountants" - https://docs.opencollective.com/help/collectives/core-contributors. I can add others like what that page instructs.

> As for the backups, it looks like yes, but actually I restart it also after kernel upgrades, so doing it twice monthly would be better than nothing. Better would be an rsync thing of /home and /etc to another machine.

Got it! Either sounds great now.

**Comment by yan12125** [comment]
(Continuation of https://github.com/lxqt/lxqt/discussions/1968#discussioncomment-581257)

Still nobody wants to be another admin for the OpenCollective project? @jubalh @luis-pereira @palinek @wxl

---

> I wouldn't mind taking care of it, but I need documentation first, and I think there's still a lot of doubt about it.

@telmo1241 For doubts, I think I've answered above. Could you also point out what kind of documentation is missing at that issue?

**Comment by telmo1241** [comment]
I wouldn't mind being an administrator, but I don't know how I do it

**Comment by yan12125** [comment]
First an OpenCollective account is needed. I can add others as admins if account names are provided. Here is an example: https://opencollective.com/standreas

**Comment by yan12125** [comment]
By the way, @stefonarch any objection if I added you as a core contributor of https://opencollective.com/lxqt?

**Comment by stefonarch** [comment]
> By the way, @stefonarch any objection if I added you as a core contributor of https://opencollective.com/lxqt?

Not at all, was about to propose that to populate it some more.

We could also make some donation to weblate and list it under "what we use it for".

**Comment by telmo1241** [comment]
![Screenshot_2021-05-07-08-21-19-291_com android chrome](https://user-images.githubusercontent.com/80489312/117420604-46ab5000-af0d-11eb-9f3e-10d4ea1c49e7.jpg)
Account created

**Comment by stefonarch** [comment]
> I prefer option 2 - I feel it not easy to estimate how much should be redirected.

My idea is that this amount shouldn't be fix, but can be decided when there are much funds.

**Comment by telmo1241** [comment]
> Prefiro a opção 2 - acho que não é fácil estimar quanto deve ser

Thank you, I'm already starting to advertise on forums about opencollective, I hope someday the money from the donation can pay someone 100% someday, meanwhile I will try to help the project the way I can

**Comment by yan12125** [comment]
> When I remember well one has also to sign up in hosted weblate to donate.

Ah, thanks for pointing out the fact. That is beyond my expectation.

> could be decided when there are much funds, or yearly 100$ or similar.

"Yearly 100$ when we have enough money" sounds better than a percentage. I just want to make expenses predictable.

> Thank you, I'm already starting to advertise on forums about opencollective

Thanks a lot for the efforts!

**Comment by yan12125** [comment]
@stefonarch Boring monthly reminder for submitting server expenses :)

Forgot about it last month :sweat:

**Comment by stefonarch** [comment]
I thought we'd do it every 3 months, no? I won't forget :)
Thinking about taking a snapshot and/or a disk image, atm it's quiet in weblate, then I should also upgrade weblate when I find the time.

**Comment by yan12125** [comment]
> I thought we'd do it every 3 months, no? I won't forget :)

Argh, I also forgot about that convention LOL

> Thinking about taking a snapshot and/or a disk image, atm it's quiet in weblate, then I should also upgrade weblate when I find the time.

No hurry :)

**Comment by yan12125** [comment]
On https://opencollective.com/lxqt, there is a line below the financial contributor Franz Geffke

> on behalf of the PantherX OS Team

Is there a similar comment or note field on weblate.org?

**Comment by stefonarch** [comment]
I didn't find the any list, but scrolling down I discover that they also are on opencollective:
https://opencollective.com/weblate
So I think it's a much better and clean solution to use this way. I see 3 options but payment looks like credit card or paypal, it would be nice to use the funds there directly, will look more into it.
![schermata-08-28-10-12](https://user-images.githubusercontent.com/10681413/131211341-88d5aee6-8dcf-465d-a92d-5fd76eb27cd7.png)

**Comment by yan12125** [comment]
> maybe as admin it's possible?

That seems the case!

![圖片](https://user-images.githubusercontent.com/1937689/131217520-505dcd3e-9a61-4ae7-a43e-2498a0e74ed1.png)

Also, both https://opencollective.com/weblate and https://opencollective.com/lxqt are hosted on "Open Source Collective". Per https://docs.opencollective.com/help/financial-contributors/collective-to-collective, there are no fees :+1:

**Comment by stefonarch** [comment]
Cool! So we have only to agree about the yearly or monthly (better IMO without fees) amount, I suppose this can be changed too. $20/month?

**Comment by yan12125** [comment]
> I suppose this can be changed too

That should be the case. At least I can cancel a monthly donation and start another.

> $20/month?

Sounds good to me. If there are no concerns, I will setup it by next week (4 Sep) - remind me if I forget :D

**Comment by stefonarch** [comment]
Fine, I will :)
We can also  mention it in "What are donations used for?"

**Comment by yan12125** [comment]
> We can also mention it in "What are donations used for?"

Done! Updated https://opencollective.com/lxqt#category-ABOUT

**Comment by yan12125** [comment]
> If there are no concerns, I will setup it by next week (4 Sep)

Done! I've also created an "update" draft at the open collective page https://opencollective.com/lxqt/updates/donations-to-open-source-projects-lxqt-relies-on - I think this is a big change and worth notifying all of our financial contributors. If that draft looks good, I will publish it in two or three days.

**Comment by stefonarch** [comment]
Looks good!

**Comment by franzos** [comment]
Excuse me if I have missed the point here ... Why would LXQt accept donations, if 50% of what appears to be the yearly budget, is in turn allocated to another project? If anything we should point out that LXQt relies on Weblate so anyone may donate to the project directly.

**Comment by stefonarch** [comment]
Donations are quite recently introduced and not even publicized much, so it's work in progress.
We have used weblate software for years for translations, so it's only natural to give something back directly.

**Comment by tsujan** [comment]
> so it's only natural to give something back directly.

I agree. LXQt translations aren't my area but I know their importance and that they wouldn't be possible without Weblate and, of course, @stefonarch's work. The number of translation PRs is amazing; without Weblate, there would be only a few each month.

**Comment by franzos** [comment]
Yes I got that but it seems like Weblate is a well funded project with premium offerings whereas LXQt is a underfunded project that might find more effective ways to use it's own donations. For example to promote LXQt (buy a banner on a popular Linux site) and encourage new contributors or even allocate money to have issues fixed by a paid developer, chosen & voted for by the LXQt project owners.

**Comment by tsujan** [comment]
> even allocate money to have issues fixed by a paid developer

I don't know about the other suggestions but this one isn't a good idea for reasons I don't have time to explain now.

**Comment by stefonarch** [comment]
* promoting: I don't think that's a good idea. LXQt is among the 6 "official" DE in debian and people can choose whatever suits them best.
* to pay for feature implementing or other coding is an approach which has to be studied very very  well in all aspects. 

Regarding weblate: I just checked and we have 166 "active" translators now on the platform.

**Comment by franzos** [comment]
> isn't a good idea for reasons I don't have time to explain now.

Fair enough. This might be difficult to handle for any number of reasons.

> promoting: I don't think that's a good idea. LXQt is among the 6 "official" DE in debian

It's not a good idea to promote LXQt because it's among the 6 official DE in Debian?

**Comment by stefonarch** [comment]
> It's not a good idea to promote LXQt because it's among the 6 official DE in Debian?

I meant with  that example: it's not an unknown DE.

**Comment by franzos** [comment]
> I meant with that example: it's not an unknown DE.

It's not "unknown" to who? From what I can gather:
- There's few if any popular distro's defaulting to LXQt
- The popular ones that do offer LXQt do little beyond listing it because it's already packaged
- There's likely a significant portion that's unaware that LXQt is even being maintained
- Overall, LXQt is likely one of the least known DE's

For example, on [Distrowatch](https://distrowatch.com/dwres.php?resource=advertise) we could promote LXQt starting from $220

Anyway, this was just _one_ idea. My primary concern here was, that you shouldn't be giving away money that has been explicit donated to help LXQt, "to give something back directly" to Weblate.

**Comment by stefonarch** [comment]
Maybe you're right about promoting - it's just not my type of beer.

For donating to weblate: I cannot see a substantial difference between paying the webserver and donating also to the software.

Issuehunt looks interesting, didn't know about. But as mentioned here already this needs a careful approach.

**Comment by franzos** [comment]
> I cannot see a substantial difference between paying the webserver and donating also to the software.

Good point but I think that's also the beauty of open source ...

- LXQt is a project that cannot afford to pay for Weblate and thus uses a self-hosted, free option
- Another business XYZ generates a revenue an can afford to pay for Weblate

Most if not all open source projects rely on some other open source project(s) to even exist and while I totally agree with "giving back", this should apply to those that **can** - otherwise it will be every open source project donating proceeds to every other open source project, and there won't be any reason for anyone to donate to a particular project anymore.

**Comment by palinek** [comment]
I, for one, see @franzos point and agree. If we really want to give back to weblate, we can buy their hosting (& administration) to get less work with maintaining of weblate translations...

**Comment by yan12125** [comment]
Not having time for all recent discussions. Some of my two cents:

* Promotion: it may be never enough. Popular commercial products are also intensively promoted. The problem is: who knows marketing? If we agree on put money on marketing, maybe @telmobss1 can give some comments
* Buying weblate service: far from affordable for the current budget because our large scale of translations. There are some discussions around https://github.com/lxqt/lxqt/issues/1495#issuecomment-671052751.
* Donating to Weblate: I'm considering pause/cancel the monthly donation temporarily if we cannot reach a consensus before the next month

**Comment by yan12125** [comment]
Another point about weblate: at first it seems donating to weblate is difficult (https://github.com/lxqt/lxqt/issues/1846#issuecomment-834600611), while it is no longer the case as weblate is also on opencollective now. Another option I mentioned earlier (https://github.com/lxqt/lxqt/issues/1846#issuecomment-834506355) can be reconsidered:

> In project descriptions, ask people to donate to Weblate instead

**Comment by franzos** [comment]
> Popular commercial products are also intensively promoted. The problem is: who knows marketing?

I suppose we'd need to look into 3 issues:
1. What do we want to achieve?
- Attract more users in general: LXQt is simple, light-weight and looks familiar
- Attract more contributors: LXQt might be easier to support; it's more modular
- Attract more developers that work on QT anyway: There are not many options other than KDE / LXQt

This will give us a better idea how the eventual advertising should look like and what sort of content it links to.

2. Where we want to advertise
- There's various sites like Distrowatch
- There are a couple of popular Linux focused magazines

3. How do we turn those ideas into reality
- If it's a banner, any number of users could contribute design ideas (I'd be happy to), which we vote for
- Someone needs to write something, for ex. a blog post that relates to the banner

I guess once we figure out 1 and 2, 3 will follow automatically.

> Buying weblate service: far from affordable for the current budget because our large scale of translations.

I also had a look at the premium offerings and none of them appear "in reach". In any case, I suppose that's the beauty: As long as LXQt remains a niche DE, we have the options to host it ourselves. As LXQt becomes more popular and more LXQt-specific issues are taken care of, we will also be able to attract more donations, which in turn might afford us to reconsider paying for Weblate services - there's no reason why LXQt shouldn't be one of the top 3 most popular DE's. It has everything going for it!

> Donating to Weblate

I think it's all about priorities and right now LXQt itself would benefit the most. Even if we don't have an immediate solution, the money doesn't run away either.

**Comment by yan12125** [comment]
Thank you very much about ideas for promotion! Could you create a new thread on https://github.com/lxqt/lxqt/discussions? "Whether money should be used for promotion or not" is OK for this issue, while "how to do promotion" is better for a new discussion, so that we can focus on money-related issues here.

**Comment by pols12** [comment]
As a potential donor, I would be pained if my money would finance advertising instead of real project costs (server loads, QA services, feature development…).

I choose my DE for its feature, not because it has nice ads.
DE should not be competitors but complement each other instead.

**Comment by franzos** [comment]
> As a potential donor, I would be pained if my money would finance advertising instead of real project costs (server loads, QA services, feature development…).

In theory I couldn't agree more. I just cannot see it happening anytime soon...

1. Which features are a priority to whom?
2. Who will implement whatever is priority?
3. Who will review the result and settle payments?

It could look very much like this: https://opencollective.com/nest

> I choose my DE for its feature, not because it has nice ads.

Yes, **you** are here already; commenting. It's great! How do we get more people to come here, and comment like you did?

**Comment by shlyakpavel** [comment]
> As a potential donor, I would be pained if my money would finance advertising instead of real project costs (server loads, QA services, feature development…).

As a current donor, I'd really love to see LXQt advertise. Unlike you, I strongly feel LXQt is not as polished as some other desktop environments and I think that's due to lack of user feedback. If LXQt becomes more popular - all of us will have a chance to get a better product.

**Comment by stefonarch** [comment]
IMHO both sides has there points. 

> I strongly feel LXQt is not as polished as some other desktop environments and I think that's due to lack of user feedback

Not only, but also lack of developers/coders, There are feature requests accepted laying around...

On another side: this " not polished" is due also on the modularity and tweakability of LXQt and that's what I like most: highly configurable and what you "hack" works always, I loved  KDE for its configuration possibilities, but got always disappointed at some point because it  lost itself among his own configuration and settings wre messed up at next login,

**Comment by stefonarch** [comment]
FYI, we have now at disposal up to 50 email aliases with our domain name, 3$/monthly at mailbox.org.
The new twitter account is opened with twitter at lxqt-project.org
There is a bus factor of 1 atm for both...

**Comment by yan12125** [comment]
Thanks! I updated https://opencollective.com/lxqt#category-ABOUT to include mailbox.org. I may co-maintain it - need time to investigate how.

**Comment by yan12125** [comment]
@stefonarch Could you create an mail alias `bot@lxqt-project.org` -> `yan12125+lxqt@gmail.com`? This mail is used to register https://github.com/LXQtBot/, and apparently GitHub blocks acess for the bot as that address is now gone: https://translate.lxqt-project.org/projects/lxqt-desktop/screengrab/#history

Regarding co-maintaining mailbox.org: I found nothing about co-maintaining a domain by multiple accounts. How about putting credentials on the server, just like what we did for Hetzner DNS?

**Comment by stefonarch** [comment]
Regarding the alias: afaik all aliases end up in lxqt-project....mailbox.org, I didn't see how to setup this. bot@lxqt-project.org is created.


> How about putting credentials on the server, just like what we did for Hetzner DNS?

Done.

**Comment by yan12125** [comment]
> afaik all aliases end up in lxqt-project....mailbox.org

No problem and thanks! It's fine as long as I can receive mails. I re-verified the email for https://github.com/LXQtBot/, and pushing translations from Weblate to GitHub should work again. There is still an alert at https://translate.lxqt-project.org/projects/lxqt-desktop/screengrab/#translations, while I guess it's just some leftovers.

**Comment by stefonarch** [comment]
Looks fine all now, thanks!

**Comment by yan12125** [comment]
Is there a conclusion on whether LXQt should continue on donating to Weblate or not? If not, I'm going to do what I mentioned above this week (https://github.com/lxqt/lxqt/issues/1846#issuecomment-915049300, https://github.com/lxqt/lxqt/issues/1846#issuecomment-915064090)

> pause/cancel the monthly donation temporarily

> In project descriptions, ask people to donate to Weblate instead

**Comment by stefonarch** [comment]
As it was my idea to give back something and we have enough funds atm I  vote for continuing.

**Comment by franzos** [comment]
Sorry but this makes no sense whatsoever. LXQt desperately needs more attention and we're not going to get anywhere if we throw the $ we have after other open source projects. If someone decides to personally donate to Weblate - sure! ... but we're collecting donations **for LXQt** and not for Weblate.

**Comment by yan12125** [comment]
Alright, apparently some concerns are not fully addressed yet. I've cancelled the monthly donation to weblate and updated descriptions at https://opencollective.com/lxqt.

**Comment by yan12125** [comment]
I'm looking forward to improving visibility of donation-related stuffs. There is no donation since March this year. If the situation continues, we may run out of money some time in 2024.

From https://github.com/lxqt/lxqt/pull/1935#issuecomment-791610593

> I think first we need to improve a little the OpenCollective page: logo, text, information about how donations are intended to use and more admins.

Looks like all are done?

**Comment by stefonarch** [comment]
> information about how donations are intended to use

This is still an issue IMO - we need very little atm. Some dev needs a new laptop ;) ?
But I could tweet about it, inserting it in lxqt-about is an option too. What was you thinking about to improve visibility?

**Comment by franzos** [comment]
> If the situation continues, we may run out of money some time in 2024.

> This is still an issue IMO - we need very little atm.

Unless there's a plan to actually put the donations to good use (to benefit LXQt), there's no reason to promote them.

**Comment by franzos** [comment]
> I think running out of money in 2 years is not a good sign

@yan12125 I wish I had your problems.

> There are already a few items listed at the bottom of https://opencollective.com/lxqt.

All I'm suggesting is that you won't have any trouble collecting $30 a month (at the current rate), but if you decide to promote the project more heavily, there should be more encouraging reasons than those listed on Open Collective.

I mean there's naturally a hard-limit to what you're going to spend on Email, Domain and Website.

So what are people donating for? Should the balance reach $2,000 before we decide to start donating to Weblate again even though people donate to LXQt, to support the LXQt team and LXQt development? I mean what's the plan here?

Maybe we could come-up with some specific goals and possibly even put a $ figure on it? Say, what if we manage to consistently collect $100, $250 or even $1000 a month? If so, what are the possible pathways to grow LXQt?

- Issue bounties?
- Promotions?
- ...

Donate for what?!

**Comment by stefonarch** [comment]
> > Some dev needs a new laptop maybe ;)
> 
> Was there a discussion? It's infeasible to buy such "expensive" items for now. If several devs agree on such items, I can add them to a paragraph like future plans on https://opencollective.com/lxqt.

We didn't discuss it yet  but limited contributions for hardware renewal for developers is a legit LXQt expense IMHO.

**Comment by palinek** [comment]
I definitely agree with @franzos. We're missing the leadership/roadmap in LXQt. Currently we're just a bunch of people doing ad-hoc enhancements and/or maintenance. There is no real vision and goals maintained.

**Comment by tsujan** [comment]
A small rant from me:

I don't think we needed someone to command us, "You shall not let LXQt 1.2.0 be in Qt5!" — porting the main components to Qt6 could have been a natural goal, as it has been [discussed](https://github.com/lxqt/lxqt/discussions/2139) since 1.0.0.

**Comment by yan12125** [comment]
GitHub Projects look good for roadmaps. I created one for Qt 6 support: https://github.com/orgs/lxqt/projects/3. Another useful one might be for Wayland support, but I'm not really interested in Wayland as of writing.

**Comment by stefonarch** [comment]
Stumbled upon https://opencollective.com/lxqt/updates/donations-to-open-source-projects-lxqt-relies-on

I didn't find any donation to weblate until now, shall we make one?

**Comment by yan12125** [comment]
> I didn't find any donation to weblate until now

There was one in 2021 https://opencollective.com/weblate/contributions/475027

> shall we make one?

Probably not. There were some objections, see discussions starting from https://github.com/lxqt/lxqt/issues/1846#issuecomment-914567371.

Also, our budget is not high. The annual income $184.92 USD ([predicted by OpenCollective](https://opencollective.com/lxqt#category-BUDGET)) is only a little bit higher than our annual spend $155.83 USD (from a quick calculation as per https://opencollective.com/lxqt#category-ABOUT).
